Doc/c-api/init.rst

```diff
@@ -1774,6 +1774,18 @@ Python-level trace functions in previous versions.
 
    The caller must hold the :term:`GIL`.
 
+.. c:function:: void PyEval_SetProfileAllThreads(Py_tracefunc func, PyObject *obj)
+
+   Like :c:func:`PyEval_SetProfile` but sets the profile function in all running threads
```

Comment:
This should probably say "all running threads belonging to the current interpreter".
Likewise for the other functions.

Reply:
Makes sense, will update